README.md

```diff
@@ -562,11 +562,13 @@ In `Connect` tab, we'll configure Sonarr to send notifications to Plex when a ne
 
 #### Give it a try
 
-Let's add a serie !
+Let's add a series !
```

Comment:
Hah, that's my poor english 😄 
Would it make more sense to call it "tv show"?

Reply:
Series and tv show are interchangeable, so in this case I think series is fine. However, when it comes to folder naming, it seems like `/tv` is the agreed upon term; this isn't an issue here though.